pytorch_lightning/loggers/base.py

```diff
@@ -310,7 +311,15 @@ def finalize(self, status: str) -> None:
         self.save()
 
     def close(self) -> None:
-        """Do any cleanup that is necessary to close an experiment."""
+        """Do any cleanup that is necessary to close an experiment.
+
+        .. deprecated:: v1.5     This method was deprecated in v1.5 and will be removed in v1.7     in favor of
+        `LightningLoggerBase.save`
```

Comment:
for the docs error, maybe there's an issue with the spacing here?

Reply:
this maybe does seem to be the issue? c707a9f [passes](https://github.com/PyTorchLightning/pytorch-lightning/runs/3563829458) but 5c77e69 somehow [does not](https://github.com/PyTorchLightning/pytorch-lightning/runs/3563863729)

Reply:
yep, just make the indentation like in all the other places and it will be good :)

Reply:
haven't been able to figure why docformatter keeps [modifying](https://github.com/PyTorchLightning/pytorch-lightning/pull/9422/commits/6d12d69876e51f394807223512d15824f8aefcfb) the correct matching format that actually [passes make-docs](https://github.com/PyTorchLightning/pytorch-lightning/runs/3569055223) :(

Reply:
oh wow, that seems to be a pretty bad bug. not sure what we can do about it, will ask around

Reply:
not sure what is the question?

Reply:
the docstring autoformatter is breaking the format

Reply:
Looks like we can't do anything about it right now, so let's merge it anyway. The message is there, only the formatting is a bit broken

Reply:
It won't get formatted if there's one more line

```python
def close():
    """Do any cleanup that is necessary to close an experiment.

    Having this line avoids the formatting.

    .. deprecated:: v1.5
        This method is deprecated in v1.5 and will be removed in v1.7.
        Please use ``something something`` instead.
    """
```

Worth opening an issue in https://github.com/myint/docformatter though

Reply:
thanks everyone! hm I added an extra line which does indeed do the trick, if it looks ok (filed https://github.com/myint/docformatter/issues/78)